screening-schema.json

```diff
@@ -453,28 +452,6 @@
                 "Unknown"
               ]
             },
-            "addressSafetySecurityUnderageUsers": {
```

Comment:
This should stay here. Instead of removing it twice, we should only remove the duplicate below, but this one remains